### PR TITLE
Fix commit section responsivness

### PR DESCRIPTION
**Pull request by Moejoe90**
image
image
image

### CODE REVIEW

**Comment by Moejoe90** [comment]
The solution is a hack using <br> on the first column to generate the desired space while reducing the bottom padding of this section. 
I think there can be a better solution, i would be up for suggestions

File: page/sass/commit_funds.scss [hunk 1]
```diff
@@ -2,8 +2,12 @@
     margin-top: $padding-between-sections;
     border-top: 1px $off-white solid;
 
+
     h3{
         font-size: 20px;
+        @media (max-width: $screen-md){
```
Comment (krzkaczor) [inline page/sass/commit_funds.scss]: why is this needed?

File: page/sass/commit_funds.scss [hunk 2]
```diff
@@ -13,9 +17,17 @@
...
         font-weight: $font-weight-more-bold;
+
     }
+.seperator {
+  @media (max-width: $screen-md){
+    margin-bottom: 55px;
+    }
+}
     .content{
+      @media (min-width: $screen-md){
```
Comment (krzkaczor) [inline page/sass/commit_funds.scss]: create one @media query and override everything there - it's easier to maintain

File: page/sass/commit_funds.scss [hunk 3]
```diff
@@ -13,15 +14,28 @@
...
+    @media (min-width: $screen-md){
         padding-left: 50px;
-        p{
-            font-size: $font-size-small
-        }
-        p strong {
-            font-weight: $font-weight-more-bold;
-        }
+    }
+    p{
```
Comment (krzkaczor) [inline page/sass/commit_funds.scss]: here
